lib/send_message.rb

```diff
@@ -0,0 +1,35 @@
+require 'rubygems' # not necessary with ruby 1.9 but included for completeness
+require 'twilio-ruby'
+require 'dotenv'
+Dotenv.load
+
+class SendMessage
+
+  def send_sms(total)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for send_sms is too high. [15.81/15] (http://c2.com/cgi/wiki?AbcMetric)<br>Metrics/MethodLength: Method has too many lines. [14/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)